### DISCUSSION TITLE
Help me please to use Wg-Easy with my k8s cluster and traefik

### DISCUSSION THREAD

**Discussion by FuriousGopher** [wg-easy/wg-easy#1172] [Q&A]
Hello everyone.
I have a task to apply vpn to k8s cluster and its something new for me =(
I have a cluster on Digital Ocean and i use Treafik like ingress controller. 
Maybe there is a video on Youtube.
Who can help me with this? 
Thank you very much

**Comment by tetuaoro** [comment]
👋🏼, I found this [wiki](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Traefik-SSL).

**Reply by FuriousGopher** [reply]
Hi, and thanks you. I saw it as well. But how i understand it's for docker compose just for https for wg-easy UI.
And i need need the way to hide my ingressRouiters by wg-easy vpn

**Reply by FuriousGopher** [reply]
> 👋🏼, I found this [wiki](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Traefik-SSL).

Still try to make it working. Maybe you know somebody who can help me ?

**Reply by tetuaoro** [reply]
> > 👋🏼, I found this [wiki](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Traefik-SSL).
> 
> Still try to make it working. Maybe you know somebody who can help me ?

Sorry no.
